src/components/UserInputBlock.tsx

```diff
@@ -29,27 +29,35 @@ interface InputFormValues {
   response: string
 }
 
-export function buildClassName(formDisabled: boolean): string {
+export function buildClassName(formDisabled: boolean, errorResponse: string | undefined): string {
   let className = 'os-form-control'
   if (formDisabled) {
     className += ' os-textarea-disabled'
   }
+  if (errorResponse !== undefined) {
+    className += ' os-wrong-answer-choice'
+  }
   return className
 }
 
 export const UserInputBlock = ({ content, prompt, ack, waitForEvent, fireEvent, buttonText, contentId, onInputSubmitted }: UserInputBlockProps): JSX.Element => {
   const [responseSubmitted, setResponseSubmitted] = useState(false)
   const contentLoadedContext = useContext(ContentLoadedContext)
+  const NON_EMPTY_VALUE_ERROR = 'Please provide valid input'
 
   const schema = Yup.object({
     response: Yup.string()
-      .trim()
-      .required('Please provide valid input')
       .max(MAX_CHARACTER_INPUT_BLOCK_LENGTH, 'Input is too long')
```

Comment:
Minor: Good call creating / using `NON_EMPTY_VALUE_ERROR`  in lieu of the in-line string definition (I think it helps with maintainability / readability even if not re-used :man_shrugging: ). It might be nice to go ahead and also define `EXCEEDED_MAX_INPUT_ERROR` and use that here.